gatsby/static/opensearch.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+    <ShortName>CKS Search</ShortName>
+    <Description>CKS Search - A search engine for Clinical Knowledge Summaries</Description>
+    <Url type="text/html" method="get" template="https://cks.nice.org.uk/Search?q={searchTerms}"/>
```

Comment:
What do you think about using an octo variable for the base URL e.g. just https://cks.nice.org.uk so we can transform it per environment to be able to search from the address bar quickly? We'll prob need the base URL elsewhere too, for example the sitemap XML could have it

Reply:
Since the other environments are not indexed it would make no difference in this scenario but it does seem right since we can use that octo variable elsewhere too.

Reply:
it's not about indexing, it's about searching - it's about how the browser interprets a website's ability to provide search - so if you start typing dev.cks in the address you'll get a "press tab" to search hint in Chrome